package.json

```diff
@@ -147,6 +150,7 @@
     "@cision/react-container-query": "1.0.0-alpha.3",
     "classnames": "^2.2.6",
     "lodash": "^4.17.15",
+    "react-intl": "^4.6.9",
```

Comment:
intentional? not seeing a new usage in the rest of the diff at least

Reply:
Good call - I was playing with some date formatting options earlier, but it didn't make this cut.